src/targets/browser/index.jsx

```diff
@@ -1,26 +1,30 @@
 /* global cozy */
 
-import 'styles'
-
 import React from 'react'
-import CozyClient, { CozyProvider } from 'cozy-client'
+import { Provider } from 'react-redux'
 import { render } from 'react-dom'
+import { hot } from 'react-hot-loader'
+
+import CozyClient, { CozyProvider } from 'cozy-client'
 import { I18n, initTranslation } from 'cozy-ui/transpiled/react/I18n'
+import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
+
 import App from 'components/App'
 import configureStore from 'store/configureStore'
-import { hot } from 'react-hot-loader'
 import { schema } from '../../helpers/doctypes'
 import manifest from '../../../manifest.webapp'
-import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
+import 'styles'
```

Comment:
```suggestion
import 'styles.styl'
```

Reply:
I think we want to import `./styles/index.styl` here, not `./styles.styl`

Reply:
So let's import ./styles/index.styl. It's better to be explicit with the extension